internal/cli/iam/organizations/invitations/describe.go

```diff
@@ -71,8 +75,10 @@ func DescribeBuilder() *cobra.Command {
 		Example: fmt.Sprintf(`  # Return the JSON-formatted details of the pending invitation with the ID 5dd56c847a3e5a1f363d424d for the organization with the ID 5f71e5255afec75a3d0f96dc:
   %s organizations invitations describe 5dd56c847a3e5a1f363d424d --orgId 5f71e5255afec75a3d0f96dc --output json`, cli.ExampleAtlasEntryPoint()),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			opts.OutWriter = cmd.OutOrStdout()
-			return opts.initStore(cmd.Context())()
+			return opts.PreRunE(
+				opts.initStore(cmd.Context()),
+				opts.InitConditionalOutput(cmd.OutOrStdout(), describeTemplateCloud, describeTemplateOnPrem),
```

Comment:
looking at this code makes me think if we just duplicated this file no if's would be needed 😅  (one for atlas and one for OnPrem)

Reply:
Just pushed a couple of commit to duplicate the files for commands + tests. 
I'm ok with this as it wouldn't be as much of a scope creep as duplicating the stores and would save some tech debt when splitting the CLIs

Only concern is the perceived size of the changes

Let me know what you think @fmenezes @wtrocki @gssbzn

Reply:
LGTM! This sounds very good. If we are concerned about the size we should have one PR that does duplication only that will make verification easier.

Reply:
To me looks better, no conditional and commands more focused.